packages/block-library/src/navigation-link/edit.js

```diff
@@ -251,6 +290,12 @@ function NavigationLinkEdit( {
 								className="wp-block-navigation-link__inline-link-input"
 								value={ link }
 								showInitialSuggestions={ true }
+								showCreateEntity={ true }
```

Comment:
Two things:

- Should this be `true` if the user does not have permissions to create pages?
- Is this redundant, if we could just infer it based on the presence of `createEntity` prop?

Reply:
Good catch. It is completely redundant as I removed the entire property yesterday in favour of relying on `createEntity` beind provided (there's even a "unit" test for that!).

You'll note that I've predicated passing in `createEntity` to `LinkControl` on whether the user has sufficient permissions.